packages/core/src/common/k8s-api/api-base.injectable.ts

```diff
@@ -3,20 +3,20 @@
  * Licensed under MIT License. See LICENSE in root directory for more information.
```

Comment:
This commit has a lot of changes to be just "fixing of integration tests".

Reply:
It is true though